### PR TITLE
initial style changes

### PR DESCRIPTION
**Pull request by bartes**
- upgrades of gems
- rubocop autocorrections
- added dev tools

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,5 +1,15 @@
-source "http://rubygems.org"
+# frozen_string_literal: true
+
+source 'http://rubygems.org'
 
 gemspec
 
+gem 'polishgeeks-dev-tools', '~> 1.4.0'
```
Comment (nijikon) [inline Gemfile]: Please remove this, it will be deprecated no need to add so many dependencies even in development. Let's use Coditsu instead.

File: .rubocop.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+AllCops:
+  TargetRubyVersion: 2.3
```
Comment (nijikon) [inline .rubocop.yml]: Why are we freezing to 2.3?

File: .rubocop.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+Metrics/ModuleLength:
```
Comment (nijikon) [inline .rubocop.yml]: Why is this needed?

File: .rubocop.yml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  Exclude:
+    - "**/*_spec.rb"
+
+Metrics/BlockLength:
```
Comment (nijikon) [inline .rubocop.yml]: Why is this needed?

File: Gemfile (hunk 1 above)
Comment (lluft) [inline Gemfile]: What is this dependency used for?

File: .rubocop.yml (hunk 2 above)
Comment (bartes) [inline .rubocop.yml]: we supports ruby from 2.2 to 2.4 .. 2.3 style syntax works without a warning for all of those

File: .rubocop.yml (hunk 3 above)
Comment (bartes) [inline .rubocop.yml]: I removed that from this PR - will check in next if we need that

File: .rubocop.yml (hunk 4 above)
Comment (bartes) [inline .rubocop.yml]: I removed that from this PR - will check in next if we need that

File: Gemfile (hunk 1 above)
Comment (bartes) [inline Gemfile]: removed - this is not needed anymore - this is a pack of tools

File: Gemfile [hunk 5]
```diff
@@ -4,4 +4,11 @@ source 'http://rubygems.org'
 
 gemspec
 
+gem 'pry', '~> 0.10.4'
```
Comment (nijikon) [inline Gemfile]: Shouldn't this be in `development` and `test` group?
